examples/manipulating_visualizing/plot_neurovault_meta_analysis.py

```diff
@@ -0,0 +1,99 @@
+"""
+NeuroVault meta-analysis of stop-go paradigm studies.
+=====================================================================
+
+This example shows how to download statistical maps from
+NeuroVault
+
+See :func:`nilearn.datasets.fetch_neurovault` documentation for more details.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+from nilearn import datasets
+
+### Define filters ############################################################
+cfilts = []  # [lambda col: col['DOI'] is not None]
+imfilts = [lambda im: "stop signal" in (im.get('cognitive_paradigm_cogatlas') or "").lower(),
```

Comment:
OK. I don't know what to conclude. I just find that the downloader is a
bit difficult to use because of these pecularities.